src/agent/structures/structures.ts

```diff
@@ -5,39 +5,45 @@ import { Pushes } from "./pushes";
 import { InitialUser, User } from "./users";
 
 export interface Geolocation {
-  country: string;
-  country_code: string;
-  region: string;
-  city: string;
-  timezone: string;
+  country?: string;
+  country_code?: string;
+  region?: string;
+  city?: string;
+  timezone?: string;
+  latitude?: string;
+  longitude?: string;
 }
 
 export interface LastPage {
   opened_at: string;
   url: string;
-  title: string;
+  title?: string;
```

Comment:
This fix should be also applied in v3.6 (it is already done in v3.3, v3.4, v3.5)